hail/python/hailtop/batch/backend.py

```diff
@@ -618,8 +634,11 @@ def __init__(
             if regions_from_conf is not None:
                 assert isinstance(regions_from_conf, str)
                 regions = regions_from_conf.split(',')
+            else:
+                regions = [ServiceBackend.default_region()]
```

Comment:
Need to update the docstring for the `regions` parameter to reflect the new default behavior.